CMakeLists.txt

```diff
@@ -17,7 +17,7 @@ if (ENABLE_STACK_DEBUG)
 endif()
 
 # For generating the bytes tables
-find_package(PythonInterp REQUIRED)
+find_package(Python REQUIRED)
```

Comment:
```suggestion
find_package(Python COMPONENTS Interpreter REQUIRED)
```

This also requires a `cmake_minimum_required` version bump to 3.12

Reply:
Do I need to change cmake_minimum_required in the CMakeLists.txt file to 3.12?

Reply:
Yes